Dockerfile

```diff
@@ -36,5 +39,11 @@ COPY . /opt/mythril
 RUN cd /opt/mythril \
   && python setup.py install
 
-COPY ./mythril/support/assets/signatures.db /root/.mythril/signatures.db
+RUN useradd -m mythril
+USER mythril
+WORKDIR /home/mythril
+
+RUN ( [ ! -z "${SOLC}" ] && set -e && for ver in $SOLC; do python -m solc.install v${ver}; done ) || true
+
+COPY --chown=mythril:mythril ./mythril/support/assets/signatures.db /root/.mythril/signatures.db
```

Comment:
--chown doesn't work for docker 17.05 which is being used, so can you use a different syntax there?

Reply:
Hmm, I would say this option has been available quite a while already. Maybe it's a good idea to update docker? This can be replaced with either 3 more lines of code (switch to `root`, `RUN chown`, switch back to `mythril`) or move this line to right after line 42 and `RUN chown`. Both solution makes dockerfile longer though and the second solution requires `solc` to be reinstalled after changing `signatures.db`. @norhh which one you prefer?

Reply:
I think it is possible to do it in one shot with `RUN sudo chown`

Reply:
@norhh To do so, the container requires `sudo` to be installed and user `mythril` is added to `sudo` group. I would argue that it's not really better than the suggested solutions as the user is given unnecessary privilege.

Reply:
yes, then we can go with the other option of RUN chown.

Reply:
@norhh which `RUN chown` solution? There are two I proposed.

Reply:
yes

Reply:
No @norhh , there are two options. You need to pick one.

"This can be replaced with either 3 more lines of code (switch to root, RUN chown, switch back to mythril) or move this line to right after line 42 and RUN chown"

Reply:
The 2nd one.

Reply:
Oops, just now I review this line again. I realized that the `signatures.db` is supposed to be copied to `mythril`'s directory and `chown` wasn't really necessary.